yarn.lock

```diff
@@ -10557,6 +10567,11 @@ react-is@^16.5.2, react-is@^16.6.3:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-16.6.3.tgz#d2d7462fcfcbe6ec0da56ad69047e47e56e7eac0"
   integrity sha512-u7FDWtthB4rWibG/+mFbVd5FvdI20yde86qKGx4lVUTWmPlSWQ4QxbBIrrs+HnXGbxOUlUzTAP/VDmvCwaP2yA==
 
+react-is@^16.7.0, react-is@^16.8.6:
```

Comment:
just curious if there is a real reason to have so many versions of this package in resolutions - maybe it could be possible to reduce all into one

Reply:
Deduped it 👍